### PR TITLE
typo: dlass --> class

### PR DESCRIPTION
**Pull request by jofranz**
Overview
----------------------------------------
Ttypo: dlass --> class

Before
----------------------------------------
Text in `/civicrm/admin/job?action=update&id=6&reset=1 `is black

After
----------------------------------------
Text in `/civicrm/admin/job?action=update&id=6&reset=1` is grey

Technical Details
----------------------------------------
fixed typo

### CODE REVIEW

File: templates/CRM/Admin/Form/Job.tpl [hunk 1]
```diff
@@ -92,7 +92,8 @@ CRM.$(function($) {
     <tr class="crm-job-form-block-scheduled-run-date">
         <td class="label">{$form.scheduled_run_date.label}</td>
         <td>{$form.scheduled_run_date.html}<br />
-            <div dlass="description">{ts}Do not run this job before this date / time. The run frequency selected above will apply thereafter.{/ts}<br />
+            <div class="description">{ts}Do not run this job before this date / time. The run frequency selected
+              above will apply thereafter.{/ts}<br />
```
Comment (demeritcowboy) [inline templates/CRM/Admin/Form/Job.tpl:96]: The dlass => class change is good but `ts` doesn't work very well when lines are split. Can you put it back on one line?

Comment (jofranz) [reply templates/CRM/Admin/Form/Job.tpl]: thanks for pointing it out! fixed!

**Comment by demeritcowboy** [comment]
jenkins test this please

**Comment by eileenmcnaughton** [comment]
Unrelated fail - merging

**Comment by eileenmcnaughton** [comment]
@jofranz thank you for the patch - can you please submit a second PR adding yourself to the contributors-key.yml file (in the root dir) so we can include you in release notes

**Comment by jofranz** [comment]
done #25314

**Pull request merged**